components/infra-proxy-service/server/proxy.go

```diff
@@ -160,6 +162,20 @@ func GetOrgAdminKeyFrom(secret *secrets.Secret) string {
 	return adminKey
 }
 
+// GetWebuiKeyFrom returns Webuikey
+func GetWebuiKeyFrom(secret *secrets.Secret) string {
```

Comment:
This work is duplicate from #6364

Reply:
Make the changes to use a single function for server and organisation key from secrets as `GetAdminKeyFrom(secret)`. After merging #6364 will keep only single function from both PRs.